grakn-engine/src/main/java/ai/grakn/engine/loader/LoaderTask.java

```diff
@@ -143,7 +143,7 @@ private void throwException(String message, Collection<InsertQuery> inserts){
      * @param e exception to log
      * @param i amount of time to sleep
      */
-    private void handleError(Throwable e, int i) {
+    private void handleError(Throwable e, long i) {
```

Comment:
In this case, the change is because `Thread.sleep` accepts a long, rather than because of our lack of faith in our code quality :P

I don't necessary know what this code does, all I know is it works with `long`, not `int`.